### PR TITLE
fix ctor D3DDepthStencilState

### PR DESCRIPTION
**Pull request by agametov**
add attribute Flags for bit operation

### CODE REVIEW

File: src/Veldrid/Graphics/Direct3D/D3DDepthStencilState.cs [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+    public class D3DDepthStencilState : DepthStencilState
     {
         private readonly Device _device;
         private readonly SharpDX.Direct3D11.DepthStencilState _deviceState;
 
         public D3DDepthStencilState(Device device, bool isDepthEnabled, DepthComparison comparison, bool isDepthWriteEnabled)
         {
             _device = device;
-            IsDepthEnabled = IsDepthEnabled;
+            IsDepthEnabled = isDepthEnabled;
```
Comment (mellinoe) [inline src/Veldrid/Graphics/Direct3D/D3DDepthStencilState.cs]: Ouch... I'm surprised I never noticed that this wasn't working.

**Comment by mellinoe** [comment]
Thanks for the fixes, @agametov , these all look good to me. Before I merge this, I am going to add a license file to the repository so it is clear that this is an open-source project. You are the first person who has submitted a patch so far 😄 .

**Comment by mellinoe** [comment]
I've added a file to the repo which clarifies the license (MIT): https://github.com/mellinoe/veldrid/blob/master/LICENSE

Let me know that you are fine with the license and I will go ahead and merge your changes. Thanks!

**Comment by agametov** [comment]
@mellinoe LGFM 👍

**Comment by mellinoe** [comment]
@agametov Sorry for the delay on this. Merging now.

**Pull request merged**